mmdet3d/models/detectors/imvoxelnet.py

```diff
@@ -0,0 +1,149 @@
+import torch
+
+from mmdet3d.core import bbox3d2result, build_anchor_generator
+from mmdet3d.models.fusion_layers.point_fusion import point_sample
+from mmdet.models import DETECTORS, build_backbone, build_head, build_neck
+from mmdet.models.detectors import BaseDetector
+
+
+@DETECTORS.register_module()
+class ImVoxelNet(BaseDetector):
+    """ImVoxelNet <https://arxiv.org/abs/2106.01178>."""
```

Comment:
r"""\`ImVoxelNet <https://arxiv.org/abs/2106.01178>\`_."""
(for the API doc automatic formatting)